src/reducers/clientReducer.js

```diff
@@ -1,7 +1,8 @@
 import { CLIENT_CHANGE } from '../constants/actionTypes'
+import { LARGE_SCREEN } from '../constants/screenSizes'
 
 const initialState = {
-  smallScreen: true,
+  screenSize: LARGE_SCREEN,
```

Comment:
Use SMALL_SCREEN as default; ideally it will trigger less reflows on mobile clients (optimally it would cause no reflow at all).